docs/development/variables/block-variables.mdx

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
@mattppal Can you look over this new doc for using block variables and let me know if it makes sense? There's a gift on the PR main page also showing the block variables in the UI.

Reply:
This is awesome. Very clear and concise. Thanks Johnson!